### PR TITLE
Allow users to !fix !resolve in issue comments

### LINKED ISSUE #944: Flag comments in PR conversation
If you have a generic concern that is not tied to a diff you should be able to add a flag (**!fix** / **!resolve**) in the pull request conversation.

### PR DESCRIPTION
**Pull request by casche**
closes #944

Work in progress 🚧

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/review.ninja/pull/1061" target="_blank">ReviewNinja

**Comment by casche** [comment]
@dfarr Issue comments will not allow us to have 'threads' out-of-the-box like PR comments. We need to add some way of identifying the issue that is creating so we know when it is closed.

What do you think of something like : [!fix] [::emoji::] [description]

> !fix 🎅 You need to format this code

...

> !fixed 🎅

File: src/server/services/pullRequest.js [hunk 1]
```diff
@@ -34,14 +34,27 @@ module.exports = {
...
+                  arg: {
+                      user: args.user,
+                      repo: args.repo,
+                      number: args.number,
+                      per_page: 100
+                  },
+                  token: args.token
+              }, function(err, issueComments) {
+                
+                var comments = prComments.concat(issueComments) || [];
```
Comment (dfarr) [inline src/server/services/pullRequest.js]: I suggest:
```
var comments = prComments.concat(issueComments) || [];

**Comment by casche** [comment]
@dfarr added a simple test case that verifies both github calls (issue comments and pr comments) are are completed

**Pull request merged**